sklearn/ensemble/tests/test_forest.py

```diff
@@ -1419,16 +1421,14 @@ def test_forest_degenerate_feature_importances():
     'max_samples, exc_type, exc_msg',
     [(int(1e9), ValueError,
       "`max_samples` must be in range 1 to 6 but got value 1000000000"),
-     (1.0, ValueError,
-      r"`max_samples` must be in range \(0, 1\) but got value 1.0"),
      (2.0, ValueError,
-      r"`max_samples` must be in range \(0, 1\) but got value 2.0"),
+      r"`max_samples` must be in range \(0, 1\] but got value 2.0"),
```

Comment:
Same comment regarding `(0.0, 1.0]` for these tests.

Reply:
fix it on https://github.com/scikit-learn/scikit-learn/pull/20159/commits/5b46672c5c1318990918c86a1f795884d7784054